.env

```diff
@@ -1 +1,2 @@
-PATH="./scripts:$PATH"
+# examples
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318
```

Comment:
I'd rather that this `.env` file was scoped to the examples that use it. I think in the future we'll want to have examples for exporting to different 3rd party providers like datadog, so we'd need different env var values in those cases.

Reply:
On such cases, we will have `OTEL_EXPORTER_XXX_ENDPOINT`s additionally?

Does `OTEL_EXPORTER_OTLP_ENDPOINT` existence make library use otlp protocol?

Reply:
I think for the different examples, we would generally need different OTEL_EXPORTER_OTLP_ENDPOINT values. I don't think there are other standardized `OTEL_EXPORTER_XXX_ENDPOINT` values currently.

Reply:
Okey 0a92377b1c863fcc3be4c1ee0601a16bfe4703b9